pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -74,12 +75,28 @@ set -o nounset # fail if variables are unset
 set -o xtrace
 
 OSD_ID="$ROOK_OSD_ID"
+CEPH_FSID=%s
 OSD_UUID=%s
 OSD_STORE_FLAG="%s"
 OSD_DATA_DIR=/var/lib/ceph/osd/ceph-"$OSD_ID"
 CV_MODE=%s
 DEVICE="$%s"
 
+# "ceph.conf" must have the "fsid" global configuration after the following Ceph's PR is merged.
+# https://github.com/ceph/ceph/commit/25655e5a8829e001adf467511a6bde8142b0a575
+cp --no-preserve=mode /etc/temp-ceph/ceph.conf /etc/ceph/ceph.conf
+python3 -c "
+import configparser
+config = configparser.ConfigParser()
+config.read('/etc/ceph/ceph.conf')
+
+if not config.has_option('global','fsid'):
+    config['global']['fsid'] = '$CEPH_FSID'
+
+with open('ceph.conf', 'w') as configfile:
```

Comment:
```suggestion
with open('/etc/ceph/ceph.conf', 'w') as configfile:
```

Reply:
Oops, sorry. I pushed the old version by mistake. Fixed.